src/plugins/platforms/freerdp/qfreerdppeer.cpp

```diff
@@ -125,102 +125,103 @@ static void rdp_peer_context_free(freerdp_peer* client, RdpPeerContext* context)
-		{KBD_INUKTITUT_LATIN, 0},
-		{KBD_CANADIAN_FRENCH_LEGACY, "fr-legacy"},
-		{KBD_SERBIAN_CYRILLIC, 0},
-		{KBD_CANADIAN_FRENCH, 0},
-		{KBD_SWISS_FRENCH, "ch"},
-		{KBD_BOSNIAN, "unicode"},
-		{KBD_IRISH, 0},
-		{KBD_BOSNIAN_CYRILLIC, 0},
-
-		{0x0000000, DEFAULT_KBD_LANG} // default
+		{KBD_ARABIC_101, "ara", "ar_MA.UTF-8"},
+		{KBD_BULGARIAN, "bg", "bg_BG.UTF-8"},
+		{KBD_CHINESE_TRADITIONAL_US, 0, 0},
+		{KBD_CZECH, "cz", 0},
+		{KBD_DANISH, "dk", "da_DK.UTF-8"},
+		{KBD_GERMAN, "de", "de_DE.UTF-8"},
+		{KBD_GREEK, "gr", 0},
+		{KBD_US, "us", "en_US.UTF-8"},
+		{KBD_SPANISH, "es", "es_ES.UTF-8"},
+		{KBD_FINNISH, "fi", "fi_FI.UTF-8"},
+		{KBD_FRENCH, "fr", "fr_FR.UTF-8"},
+		{KBD_HEBREW, "il", 0},
+		{KBD_HUNGARIAN, "hu", "hu_HU.UTF-8"},
+		{KBD_ICELANDIC, "is", 0},
+		{KBD_ITALIAN, "it", "it_IT.UTF-8"},
+		{KBD_JAPANESE, "jp", 0},
+		{KBD_KOREAN, "kr", 0},
+		{KBD_DUTCH, "nl", "nl_NL.UTF-8"},
+		{KBD_NORWEGIAN, "no", "no_NO.UTF-8"},
+		{KBD_POLISH_PROGRAMMERS, 0, 0},
+//		{KBD_PORTUGUESE_BRAZILIAN_ABN0416, 0, 0},
+		{KBD_ROMANIAN, 0, 0},
+		{KBD_RUSSIAN, "ru", "ru_RU.UTF-8"},
+		{KBD_CROATIAN, 0, 0},
+		{KBD_SLOVAK, 0, 0},
+		{KBD_ALBANIAN, 0, 0},
+		{KBD_SWEDISH, 0, 0},
+		{KBD_THAI_KEDMANEE, 0, 0},
+		{KBD_TURKISH_Q, 0, 0},
+		{KBD_URDU, 0, 0},
+		{KBD_UKRAINIAN, 0, 0},
+		{KBD_BELARUSIAN, 0, 0},
+		{KBD_SLOVENIAN, 0, 0},
+		{KBD_ESTONIAN, "ee", 0},
+		{KBD_LATVIAN, 0, 0},
+		{KBD_LITHUANIAN_IBM, 0, 0},
+		{KBD_FARSI, 0, 0},
+		{KBD_VIETNAMESE, 0, 0},
+		{KBD_ARMENIAN_EASTERN, 0, 0},
+		{KBD_AZERI_LATIN, 0, 0},
+		{KBD_FYRO_MACEDONIAN, 0, 0},
+		{KBD_GEORGIAN, 0, 0},
+		{KBD_FAEROESE, 0, 0},
+		{KBD_DEVANAGARI_INSCRIPT, 0, 0},
+		{KBD_MALTESE_47_KEY, 0, 0},
+		{KBD_NORWEGIAN_WITH_SAMI, 0, 0},
+		{KBD_KAZAKH, 0, 0},
+		{KBD_KYRGYZ_CYRILLIC, 0, 0},
+		{KBD_TATAR, 0, 0},
+		{KBD_BENGALI, 0, 0},
+		{KBD_PUNJABI, 0, 0},
+		{KBD_GUJARATI, 0, 0},
+		{KBD_TAMIL, 0, 0},
+		{KBD_TELUGU, 0, 0},
+		{KBD_KANNADA, 0, 0},
+		{KBD_MALAYALAM, 0, 0},
+		{KBD_MARATHI, 0, 0},
+		{KBD_MONGOLIAN_CYRILLIC, 0, 0},
+		{KBD_UNITED_KINGDOM_EXTENDED, 0, 0},
+		{KBD_SYRIAC, 0, 0},
+		{KBD_NEPALI, 0, 0},
+		{KBD_PASHTO, 0, 0},
+		{KBD_DIVEHI_PHONETIC, 0, 0},
+		{KBD_LUXEMBOURGISH, 0, 0},
+		{KBD_MAORI, 0, 0},
+		{KBD_CHINESE_SIMPLIFIED_US, 0, 0},
+		{KBD_SWISS_GERMAN, 0, 0},
+		{KBD_UNITED_KINGDOM, "gb", "en_GB.UTF-8"},
+		{KBD_LATIN_AMERICAN, "latam", "es_MX.UTF-8"},
+		{KBD_BELGIAN_FRENCH, "be", "fr_BE.UTF-8"},
+		{KBD_BELGIAN_PERIOD, "be", "fr_BE.UTF-8"},
+		{KBD_PORTUGUESE, 0, 0},
+		{KBD_SERBIAN_LATIN, 0, 0},
+		{KBD_AZERI_CYRILLIC, 0, 0},
+		{KBD_SWEDISH_WITH_SAMI, 0, 0},
+		{KBD_UZBEK_CYRILLIC, 0, 0},
+		{KBD_INUKTITUT_LATIN, 0, 0},
+		{KBD_CANADIAN_FRENCH_LEGACY, "fr-legacy", "fr_CA.UTF-8"},
+		{KBD_SERBIAN_CYRILLIC, 0, 0},
+		{KBD_CANADIAN_FRENCH, 0, 0},
+		{KBD_SWISS_FRENCH, "ch", "fr_CH.UTF-8"},
+		{KBD_BOSNIAN, "unicode", 0},
+		{KBD_IRISH, 0, 0},
+		{KBD_BOSNIAN_CYRILLIC, 0, 0},
+
+		{0x0000000, DEFAULT_KBD_LANG, 0} // default
 };
```

Comment:
Not your code, but figuring that I should have used `nullptr` instead of 0 in this array

Reply:
Agreed